Misc/NEWS.d/next/Library/2020-08-29-16-07-36.bpo-41662.Mn79zh.rst

```diff
@@ -0,0 +1 @@
+Fixed two rare bugs in binding parameters in the :mod:`sqlite3` module.
```

Comment:
Unfortunately, this doesn't give readers any meaningful information so if you think it's not easy to explain what we are fixing here, I'd say it's OK to not add a NEWS entry.

Reply:
Does it look better now? I added two separate entries for two bugs.